### PR TITLE
update instructions how to get started (installation of your Mac) (DEV-2215)

### PR DESCRIPTION
**Pull request by jnussbaum**
resolves DEV-2215

### CODE REVIEW

File: docs/developers/getting-started.md [hunk 1]
```diff
@@ -1,213 +1,274 @@
...
 ```
 
-This will install [bazelisk](https://github.com/bazelbuild/bazelisk) which is
-a wrapper to the `bazel` binary. It will, when the `bazel` command ir run,
-automatically install the supported Bazel version, defined in the `.bazelversion`
-file in the root of the `knora-api` repository.
+You can then look at the data in a browser at the address [http://0.0.0.0:4200/](http://0.0.0.0:4200/).
+
+Feel free to modify `rosetta.json` and `rosetta.xml` to your liking,
+restart DSP-API,
```
Comment (flavens) [inline docs/developers/getting-started.md:95]: I would add the link to the DSP-API doc

File: docs/developers/getting-started.md [hunk 2]
```diff
@@ -1,213 +1,274 @@
...
+create `rosetta.json` and upload `rosetta.xml` again,
```
Comment (flavens) [inline docs/developers/getting-started.md:96]: I would add the link to the DSP-TOOLS doc

**Review by flavens — CHANGES_REQUESTED** [review]
Here are my suggestions for you. I would use the same turn of phrase everywhere and add colons at the end of each sentence that precedes a command line.

File: docs/developers/getting-started.md [hunk 3]
```diff
@@ -1,213 +1,274 @@
...
-### Local Development Environment
+Follow the instructions on this page to install the DSP software stack on your local machine.
 
-At the DaSCH, the principal development environment is [Apple macOS](https://www.apple.com/macos).
+The basic components of DSP are:
 
-Each developer machine should have the following prerequisites installed:
+| Component | Description                                                                                    |
+| --------- | ---------------------------------------------------------------------------------------------- |
+| DSP-API   | The core of our software stack, the RDF database that offers an access via API                 |
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:12]: I would go with this suggestion from Flavie: The core of our software stack is the RDF database that offers access via API.

File: docs/developers/getting-started.md [hunk 4]
```diff
@@ -1,213 +1,274 @@
...
+| DSP-APP   | The web application that allows you to view and edit data in your browser                      |
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:13]: This is the web application that allows you to view, add and edit data to your research project

File: docs/developers/getting-started.md [hunk 5]
```diff
@@ -1,213 +1,274 @@
...
 
-- Docker Desktop: <https://www.docker.com/products/docker-desktop>
-- Homebrew: <https://brew.sh>, which can be used to install:
-  - `git`
-  - `expect`
-  - `sbt`
-  - `python` (Python 3)
-  - `node`
+The DSP software can only be run on macOS and Linux.
+Unfortunately, Windows is not supported.
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:17]: Windows is not supported.
-> remove "unfortunately"

File: docs/developers/getting-started.md [hunk 6]
```diff
@@ -1,213 +1,274 @@
...
 
-### Java AdoptOpenJDK 11
+This page is divided in 2 sections:
 
-To install, follow these steps:
+- Researchers, data stewards, and other users can take the shortcut described in the
+  [first paragraph](#install-dsp-instructions-for-users)
+- Developers must install everything, which is described in the
+  [second paragraph](#install-dsp-instructions-for-developers)
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:24]: As Flavie mentioned already, for me it is not clear which are the first and the second paragraph. Where do they start? I would like to see how they are named without having to click on the link.

File: docs/developers/getting-started.md [hunk 7]
```diff
@@ -1,213 +1,274 @@
...
+
+
+
+
+## Install DSP: Instructions for users
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:29]: Install DSP: Instructions for Researchers and Data Stewards
-> I would call it exactly the same as in text

File: docs/developers/getting-started.md [hunk 8]
```diff
@@ -1,213 +1,274 @@
...
-  - `python` (Python 3)
-  - `node`
+The DSP software can only be run on macOS and Linux.
+Unfortunately, Windows is not supported.
 
-### Java AdoptOpenJDK 11
+This page is divided in 2 sections:
 
-To install, follow these steps:
+- Researchers, data stewards, and other users can take the shortcut described in the
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:21]: Which other users? I would remove that and only talk about researchers and data stewards.

File: docs/developers/getting-started.md [hunk 9]
```diff
@@ -1,213 +1,274 @@
...
+
+
+
+
+## Install DSP: Instructions for users
+
+Researchers and data stewards who want to use DSP to manage their data can get started quite quickly.
+In order to run DSP-TOOLS on your computer, it is enough if you install the following software:
+
+### XCode command line tools
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:34]: Please add some numbers to indicate that this is the first step: 1. XCode command line tools

File: docs/developers/getting-started.md [hunk 10]
```diff
@@ -1,213 +1,274 @@
...
 ```bash
-brew tap AdoptOpenJDK/openjdk
-brew cask install AdoptOpenJDK/openjdk/adoptopenjdk11
+xcode-select --install
 ```
 
-To pin the version of Java, you can add this environment variable to your startup script (bashrc, etc.):
+You will be asked in a prompt if you want to install the command line developer tools. Click "Install".
+
+### Docker Desktop
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:47]: Please add some numbers to indicate that this is the second step: 2.  Docker Desktop

File: docs/developers/getting-started.md [hunk 11]
```diff
@@ -1,213 +1,274 @@
...
+
+DSP-API and DSP-APP are shipped as Docker containers.
+Install Docker Desktop from [https://www.docker.com/products/docker-desktop/](https://www.docker.com/products/docker-desktop/).
+Make sure to use the correct chip architecture (Intel versus Apple M1 chip).
+
+### Python
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:53]: Please add some numbers to indicate that this is the third step: 3. Python

File: docs/developers/getting-started.md [hunk 12]
```diff
@@ -1,213 +1,274 @@
...
+
+DSP-TOOLS is written in Python.
+Even if there is a pre-installed version of Python on your computer,
+we recommend installing it separately.
+It can be downloaded from [https://www.python.org/downloads/](https://www.python.org/downloads/).
+
+### DSP-TOOLS
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:60]: Please add some numbers to indicate that this is the fourth step: 4. DSP-TOOLS

File: docs/developers/getting-started.md [hunk 13]
```diff
@@ -1,213 +1,274 @@
...
-You can also use [jEnv](https://www.jenv.be/) to use different versions of Java for different things.
+From time to time, you have to upgrade DSP-TOOLS to the latest version.
+This is done in a Terminal window with the command
 
-### Bazel build tools
+```bash
+pip3 install --upgrade dsp-tools
+```
+
+### DSP-API and DSP-APP
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:75]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 14]
```diff
@@ -1,213 +1,274 @@
...
 
-To install, follow these steps:
+Follow [these instructions](https://docs.dasch.swiss/latest/DSP-TOOLS/start-stack/)
+to start up DSP-API and DSP-APP.
+
+### Try out our test project *Rosetta*
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:80]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 15]
```diff
@@ -1,213 +1,274 @@
...
+
+
+
+
+## Install DSP: Instructions for developers
+
+If you want to contribute to the development of DSP, you have to install a few more dependencies.
 
-### Clone DSP-API from GitHub
+### XCode command line tools
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:108]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 16]
```diff
@@ -1,213 +1,274 @@
...
 ```bash
-git clone https://github.com/dasch-swiss/dsp-api.git
+xcode-select --install
 ```
 
-This will install the directory `dsp-api` with subdirectories in the chosen directory.
+You will be asked in a prompt if you want to install the command line developer tools. Click "Install".
 
-### Bazel Commands
+### Homebrew
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:120]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 17]
```diff
@@ -1,213 +1,274 @@
...
+DSP-API and DSP-APP are shipped as Docker containers.
+[Docker Desktop](https://www.docker.com/products/docker-desktop/) can be installed with
 
 ```bash
-make init-db-test
+brew install --cask docker
 ```
 
-Then start the DSP stack:
+### Python
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:137]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 18]
```diff
@@ -1,213 +1,274 @@
...
-# build webapi
-bazel build //webapi/...
-
-# run all webapi tests
-bazel test //webapi/...
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 ```
 
-### Build the docker image
+### Docker Desktop
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:128]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 19]
```diff
@@ -1,213 +1,274 @@
...
 ```bash
-make stack-up
+brew install python
 ```
 
-This should start the complete Knora stack consisting of
-Fuseki, DSP-API, Redis, and Sipi.
-If everything worked properly, the Dashboard in Docker Desktop should show
-those containers running.
+### Git
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:148]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 20]
```diff
@@ -1,213 +1,274 @@
...
+and hosted on [GitHub](https://github.com/).
+Git comes with the XCode command line tools, but we recommend installing it separately, with
 
 ```bash
-make stack-down
+brew install git
 ```
 
-Please see the `Makefile` for other useful `make` targets.
+### OpenJDK 17
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:158]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 21]
```diff
@@ -1,213 +1,274 @@
...
+sdk install java 17.0.5-tem
+```
 
-1. Add Bazel Plugin and Project to IntelliJ
-    1. The latest version of the [Bazel plugin](https://plugins.jetbrains.com/plugin/8609-bazel/versions)
-       supports only IntelliJ upto version `2020.01.04`. After you make sure to
-       run this version of IntelliJ, install the plugin from inside IntelliJ.
-    1. Click on `File -> Import Bazel Project` and select twice `next`.
-    1. Uncomment the `Scala` language and click `Finish`.
+### sbt
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:186]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 22]
```diff
@@ -1,213 +1,274 @@
...
 
-    ```bash
-    bazel test //webapi/src/test/scala/org/knora/webapi/e2e/v1:SearchV1R2RSpec
-    ```
+```bash
+brew install sbt
+```
 
-1. Run single spec and only tests containing `gaga` in the description
+### Node.js, npm, and Angular
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:195]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 23]
```diff
@@ -1,213 +1,274 @@
...
+```bash
+npm install -g @angular/cli
+```
 
-To reconcile this, Bazel has an option called the *workspace status command*.
-This command is run outside of any sandboxes on the local machine, so it can
-access anything about your source control, OS, or anything else you might want
-to include. It then dumps its output into `bazel-out/volatile-status.txt`, which
-you can use (and certain language rulesets provide support for accessing from code).
+### yarn
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:213]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 24]
```diff
@@ -1,213 +1,274 @@
...
+DSP-APP uses [yarn](https://yarnpkg.com/getting-started/install) as a package manager.
+Install yarn with
 
 ```bash
-build --workspace_status_command=tools/buildstamp/get_workspace_status --stamp=yes
+brew install yarn
 ```
 
-Any line added to `.bazelrc` is invoked on each corresponding command.
+### DSP-API and DSP-APP
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:222]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 25]
```diff
@@ -1,213 +1,274 @@
...
 ```bash
-query --package_path %workspace%:[PATH TO BAZEL]/base_workspace # set the path to the bazel binary
+make stack-down-delete-volumes
 ```
 
-Run bazel query inside your project directory, asking it to search for all dependencies
-of //:main (or however the label is to your target of interest):
+Please see the `Makefile` for other useful `make` targets.
+
+### DSP-TOOLS
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:250]: Please add some numbers to indicate which step that is.

File: docs/developers/getting-started.md [hunk 26]
```diff
@@ -1,213 +1,274 @@
...
+
+DSP-TOOLS is installed via Terminal: Open a Terminal window and type
 
 ```bash
-export JAVA_HOME=`/usr/libexec/java_home -v 11`
+pip3 install dsp-tools
 ```
 
-You can also use [jEnv](https://www.jenv.be/) to use different versions of Java for different things.
+From time to time, you have to upgrade DSP-TOOLS to the latest version.
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:68]: I would add something like that: Please check every time before using DSP-TOOLS that you have the newest version of DSP-Tools running.

File: docs/developers/getting-started.md [hunk 27]
```diff
@@ -1,213 +1,274 @@
...
-automatically install the supported Bazel version, defined in the `.bazelversion`
-file in the root of the `knora-api` repository.
+You can then look at the data in a browser at the address [http://0.0.0.0:4200/](http://0.0.0.0:4200/).
+
+Feel free to modify `rosetta.json` and `rosetta.xml` to your liking,
+restart DSP-API,
+create `rosetta.json` and upload `rosetta.xml` again,
+and see how the data changes in DSP-APP.
+
+That's it, already!
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:99]: As Flavie mentioned, I would reformulate it as well.

File: docs/developers/getting-started.md [hunk 28]
```diff
@@ -1,213 +1,274 @@
...
+
+
+
+
+## Install DSP: Instructions for developers
+
+If you want to contribute to the development of DSP, you have to install a few more dependencies.
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:106]: I would rephrase it something like: ...., you need to install a few additional dependencies.

File: docs/developers/getting-started.md [hunk 29]
```diff
@@ -1,213 +1,274 @@
...
 
-    ```bash
-    bazel run //webapi:main_library_repl
-    ```
+```bash
+brew install node
+```
 
-### Build stamping
+Angular can be installed with
```
Comment (danielasubotic) [inline docs/developers/getting-started.md:207]: Generally, I would use more the active voice than the passive voice and use it consistently throughtout the guide. Here it is passive and if you go to the yarn section, it is: Install yarn with

**Review by danielasubotic — CHANGES_REQUESTED** [review]
see the comments

File: docs/developers/getting-started.md (hunk 8 above)
Comment (jnussbaum) [reply docs/developers/getting-started.md:21]: I think it's important to address the "other users" explicitly. Not everyone who reads our docs is either a researcher or a data steward. There might be other groups of users.

**Comment by jnussbaum** [comment]
@flavens @danielasubotic : Thanks for your feedback 👍  I implemented a lot of it. How do you like it now?

**Review by flavens — APPROVED** [review]
